packages/webrtc-client/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@fishjam-cloud/webrtc-client",
+  "version": "0.9.0",
```

Comment:
Maybe we should start versioning from `0.1.0`

Reply:
why should we if the whole monorepo shares the same version?